node/cmd/guardiand/adminverify.go

```diff
@@ -14,6 +14,7 @@ import (
 	"google.golang.org/protobuf/encoding/prototext"
 
 	nodev1 "github.com/certusone/wormhole/node/pkg/proto/node/v1"
+	"github.com/status-im/keycard-go/hexutils"
```

Comment:
Is the certusone reference in line 16 still valid?

Reply:
I see it all over the imports in the guardian, so I assume so?